README.md

```diff
@@ -115,9 +112,9 @@ before diving in here. That will introduce you to the various components used.
    
             docker exec -it airlock_ceph_1 bash
             
-   3. Set the `testuser` to be a `system` user:
+   3. Set the `ceph-admin` to be a `system` user:
    
-            radosgw-admin user modify --uid testuser --system
+            radosgw-admin user modify --uid ceph-admin --system
```

Comment:
Here I think step with testuser is ok.
This step is only required if we are not able to create system user by proxy. If we can we can remove whole point

Reply:
yes, airlock is creating users and before the change user has been created with no system permission, hence this step described as manual.
each user is created with name taken from request - not ceph-admin